src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/EHInfo.cs

```diff
@@ -132,11 +132,11 @@ public void WriteTo(TextWriter writer, int methodRva, bool dumpRva)
             switch (Flags & CorExceptionFlag.COR_ILEXCEPTION_CLAUSE_KIND_MASK)
             {
                 case CorExceptionFlag.COR_ILEXCEPTION_CLAUSE_NONE:
-                    writer.Write($" CATCH: {0}", ClassName ?? "null");
+                    writer.Write(" CATCH: {0}", ClassName ?? "null");
```

Comment:
```suggestion
                    writer.Write($" CATCH: {ClassName ?? "null"}");
```
The rest of the method uses interpolated strings. Keep using interpolated string here?

Reply:
I was torn which way is better; the rest of the code doesn't use complex expressions in the interpolations. I'm fine with either way.